subprojects/android-test/test-app/build.gradle.kts

```diff
@@ -178,6 +178,33 @@ extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
             }
         }
     }
+
+    configurationsContainer.register("uiDebug") {
+        tryToReRunOnTargetBranch = false
+        reportSkippedTests = false
+        rerunFailedTests = false
+        reportFlakyTests = false
+        // uncomment after 2020.3.6 release (MBS-8050)
+        // debugMode = true
```

Comment:
Can you please share your ideas about perfect naming for this parameter? Right now I see only criticism but without any suggestions.

Reply:
And why debugMode should be in dynamicParams? `uiDebug` configuration was create exactly for test debugging. So `debugMode` should be always `true`.

Reply:
I don't like that `debugMode` said me nothing about what I will debug.
Could we rename it to `enableDeviceDebug`

Reply:
About dynamic.
I don't like that We have a specific flag that works only with concrete target specification. I think we need abstract instrumentationConfiguration. It will help us to define a specific type-safe configuration. And We could extract your build script code to plugin and client will set only a few of parameters

Reply:
I've renamed `debugMode` to `enableDeviceDebug`.

About the convenient way of running tests in debug mode. We discussed it internally that this configuration is intended for our internal usage. For our customers we will design a proper and handy way to run test with a debugger.